src/experiments/single_dptest.sh

```diff
@@ -1,37 +1,31 @@
 #!/bin/bash
-
-run_id="$1"
+run_path=$1 # Path storing checkpoints
 ckpt_name=$2
 head_name=$3
-prefix="/mnt/workspace/public/multitask/training_exps/" # may need update
-outdir="/mnt/workspace/public/multitask/eval_scripts/temp_frz_model" # may need update
-run_pth="${prefix}${run_id}"
+outdir="/mnt/workspace/public/multitask/eval_scripts/temp_frz_model/" # may need update
+run_id=$(basename $run_path) # folder name as id
 testfile="/mnt/workspace/public/multitask/eval_scripts/test_files/${head_name}.txt" # may need update
```

Comment:
these should be parameterized,  can provide default values in python interface

Reply:
yes, maybe we can pass a `workdir` param through python.